superdesk/publish/formatters/ninjs_formatter.py

```diff
@@ -322,6 +322,14 @@ def _format_related(self, article, subscriber):
                     item = orig_item.copy()
 
                 item = self._transform_to_ninjs(item, subscriber, recursive=False)
+
+                # Keep original POI and get rid of all other POI.
+                renditions = item.get('renditions')
+                if renditions:
+                    for rendition in list(renditions):
+                        if rendition != 'original' and renditions[rendition].get('poi'):
```

Comment:
I think there might be cases when `renditions[rendition]` is `None` if image is too small for some custom one so it would be good to check it